scripts/two_line_bash.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#
+#
+# This was casuing issues in tmux and always showed an "catkin_mil_ws", I'm sure that was intended
```

Comment:
Tmux didn't behave correctly after going full screen and back - @whispercoros can you explain why the catkin mil_ws was added?